### PR TITLE
address some minor errors in the quick-start.md

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

**Comment by goneri** [comment]
I signed it

File: site/content/docs/user/quick-start.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ shown [here](https://golang.org/doc/code.html#GOPATH) if you encounter the error
 `kind: command not found` after installation.
 
 **NOTE**: `go get` should not be run from a Go [modules] enabled project directory,
-as go get inside a modules enabled project updates dependencies / behaves differently. Try for example `cd $HOME` first.
+as go get inside a module enabled project updates dependencies / behaves differently. Try for example `cd $HOME` first.
```
Comment (BenTheElder) [inline site/content/docs/user/quick-start.md:31]: this one was actually correct, AIUI, we're talking about the collective enablement of "[Go] modules", not about a _module_.

Comment (BenTheElder) [reply site/content/docs/user/quick-start.md:31]: see e.g. https://www.jetbrains.com/help/go/create-a-project-with-go-modules-integration.html
> A new Go modules project already has Go modules enabled. If you pulled your Go modules project from Github, you need to enable Go modules manually.

**Comment by BenTheElder** [comment]
thanks! one comment

File: site/content/docs/user/quick-start.md (hunk 1 above)
Comment (goneri) [reply site/content/docs/user/quick-start.md:31]: Thanks, I see.

**Comment by BenTheElder** [comment]
unfortunately the CNCF's CLA robot doesn't seem to think the CLA is signed, and the kubernetes org(s) require this 😞 
if you run into issues, they have a help-desk email address, for what it's worth. https://github.com/kubernetes/community/blob/master/CLA.md#troubleshooting

**Comment by goneri** [comment]
> unfortunately the CNCF's CLA robot doesn't seem to think the CLA is signed, and the kubernetes org(s) require this 
> if you run into issues, they have a help-desk email address, for what it's worth. https://github.com/kubernetes/community/blob/master/CLA.md#troubleshooting

Thanks, indeed, it's a rather annoying process. But well, this time it looks good :-).

File: site/content/docs/user/quick-start.md [hunk 2]
```diff
@@ -388,7 +388,7 @@ You can configure kind to use a proxy using one or more of the following [enviro
 * `NO_PROXY` or `no_proxy`
 
 **Note**: If you set a proxy it would be used for all the connection requests.
-It's important that you define what addresses doesn't need to be proxied with the NO_PROXY variable, typically you should avoid to proxy your docker network range `NO_PROXY=172.17.0.0/16`
+It's important that you define what addresses don't need to be proxied with the NO_PROXY variable, typically you should avoid to proxy your docker network range `NO_PROXY=172.17.0.0/16`
```
Comment (BenTheElder) [inline site/content/docs/user/quick-start.md:391]: aside: we may not need this cc @aojea

**Comment by BenTheElder** [comment]
/lgtm
/approve
thanks!

File: site/content/docs/user/quick-start.md (hunk 2 above)
Comment (aojea) [reply site/content/docs/user/quick-start.md:391]: indeed

https://github.com/kubernetes-sigs/kind/blob/1b4b217cfd74fc2181ce0d9888a80ed1a4cd0161/pkg/cluster/internal/providers/docker/provision.go#L288-L302

**Pull request merged**